libraries/stencil/src/components.d.ts

```diff
@@ -13,17 +13,45 @@ export namespace Components {
     }
     interface ComponentWithDeclarativeEvent {
         "camelHandled": boolean;
+        /**
```

Comment:
Oh good, I was seeing this popup in a couple of places, now it will be quite again. 🙇🏼